src/Extend/Notification.php

```diff
@@ -16,7 +16,16 @@ class Notification implements ExtenderInterface
 {
     private $blueprints = [];
     private $serializers = [];
+    private $drivers = [];
 
+    /**
+     * @param string $blueprint The ::class attribute of the blueprint class.
+     *                          This blueprint should implement \Flarum\Notification\Blueprint\BlueprintInterface.
+     * @param string $serializer The ::class attribute of the serializer class.
+     *                           This serializer should extend from \Flarum\Api\Serializer\AbstractSerializer.
+     * @param array $channelsEnabledByDefault The channels enabled by default for this notification type
```

Comment:
maybe "names of the channels..."

Reply:
Also, might be good to standardize wording to "drivers"

Reply:
Yes, there are some places where it says channels and others where it says drivers. Would be confusing for those unfamiliar with the code.